src/main/resources/db/migration/V2023.06.19.0001__add_goal_step_tables.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE goal
+(
+    id          VARCHAR(36) PRIMARY KEY,
+    reference   VARCHAR(36)  NOT NULL,
+    title       VARCHAR(512) NOT NULL,
+    review_date DATE         NOT NULL,
+    category    VARCHAR(50)  NOT NULL,
+    status      VARCHAR(50)  NOT NULL,
+    notes       TEXT,
+    created_at  TIMESTAMP(3) WITH TIME ZONE NOT NULL,
+    updated_at  TIMESTAMP(3) WITH TIME ZONE NOT NULL
```

Comment:
May as well `created_by` and `updated_by`?

Reply:
Done